block_conv.go

```diff
@@ -233,6 +237,7 @@ func (a *Accessory) MarshalJSON() ([]byte, error) {
 
 // UnmarshalJSON implements the Unmarshaller interface for Accessory, so that any JSON
 // unmarshalling is delegated and proper type determination can be made before unmarshal
+// Note: datetimepicker is not supported in Accessory
```

Comment:
[Question] Is it correct to say that this is because datetimepicker does not work with Section?

Reply:
I believe so, and the builder complains about this https://app.slack.com/block-kit-builder/T3E1TGDM3#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22Pick%20a%20date%20for%20the%20deadline.%22%7D,%22accessory%22:%7B%22type%22:%22datetimepicker%22,%22action_id%22:%22datepicker-action%22%7D%7D%5D%7D

I assume that the space is not permitted for the layout. I hope Slack supports grid layout like Teams in future. Currently, everything seems to be limited to 2 columns at most.

Reply:
I might be wrong on this. Let me check one more time. Looks like it is supported
https://github.com/slackapi/java-slack-sdk/commit/4176b912895c57e99de5d3a8ee691807801159f2#diff-212bb028a409f158a94e9af3431b61e080b8bec0e72711ba99dc294e02fd5084L2866

Reply:
Working on it. Should be done soon. Thanks for asking this. The assumption needs to be tested.

Reply:
Hmm.  I tested it out, and my assumption is right.
```
0 = {string} "[ERROR] failed to match all allowed schemas [json-pointer:/view]"
1 = {string} "[ERROR] unsupported type: datetimepicker [json-pointer:/view/blocks/8/accessory/type]"
```
datetimepicker won't work in Section.